### PR TITLE
Issue #22- [M8]-[Replace this use of System.out or System.err by a logger.]

### PR DESCRIPTION
**Pull request by katikireddy622**
**What's in the PR**

Replaced System.out & System.err with Loggers

### CODE REVIEW

File: .sonarcloud.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (munterkalmsteiner) [inline .sonarcloud.properties]: Do not include sonarqube files in the repository.

File: inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplQualifierIntegrationTest.java [hunk 2]
```diff
@@ -108,7 +112,7 @@ public void setUp() throws Exception
         sut.createConcept(kb, concept);
         sut.createProperty(kb, property);
         conceptHandle = concept.toKBHandle();
-        propertyHandle = property.toKBHandle();
+        property.toKBHandle();
```
Comment (munterkalmsteiner) [inline inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplQualifierIntegrationTest.java]: This change is not related to logging.

**Review by munterkalmsteiner — CHANGES_REQUESTED** [review]
Looks good, except two issues (see comments).

**Review by katikireddy622 — COMMENTED** [review]
sonarcloud.properties file deleted, not relevant changes to logging has been resolved and merge conflicts are resolved.

**Review by munterkalmsteiner — APPROVED** [review]
Looks good, thanks!

**Pull request closed without merging**